facebook/OpenBIC
# fby35: gl: Correct PWRGD_CPU_LVC3's signal edge

Summary:
    - Correct PWRGD_CPU_LVC3's signal edge from rising edge to both edge.

Test plan:
    - Build code: Pass
    - Check log after power on/off: Pass
    - Access host console: Pass

Test log:
    - Check log after power on/off:
        root@bmc-oob:~# power-util slot1 off
        Powering fru 1 to OFF state...
        root@bmc-oob:~# power-util slot1 status
        Power status for fru 1 : OFF
        root@bmc-oob:~# power-util slot1 on
        Powering fru 1 to ON state...
        root@bmc-oob:~# power-util slot1 status
        Power status for fru 1 : ON
        root@bmc-oob:~# log-util slot1 --print
        2018 Mar 13 23:49:45 log-util: User cleared all logs
        2018 Mar 13 23:52:07 log-util: User cleared FRU: 1 logs
        1    slot1    2018-03-13 23:52:39    power-util       SERVER_POWER_OFF successful for FRU: 1
        1    slot1    2018-03-13 23:52:40    gpiod            FRU: 1, System powered OFF
        0    all      2018-03-13 23:52:54    gpiod            NIC Power is set to VAUX
        1    slot1    2018-03-13 23:53:18    gpiod            FRU: 1, System powered ON
        0    all      2018-03-13 23:53:18    gpiod            NIC Power is set to VMAIN
        1    slot1    2018-03-13 23:53:20    power-util       SERVER_POWER_ON successful for FRU: 1
    - Access host console:
        root@bmc-oob:~# sol-util slot1

        ------------------TERMINAL MULTIPLEXER---------------------
        CTRL-l ?   : Display help message.
        CTRL-l x : Terminate the connection.
        /var/log/mTerm_slot1.log : Log location
        CTRL-l + b : Send Break

        -----------------------------------------------------------

        [root@FBK8_221107 ~]#
